src/main.cpp

```diff
@@ -111,6 +113,7 @@ int main ( int argc, char** argv )
         {
             bUseGUI = false;
             tsConsole << "- no GUI mode chosen" << endl;
+            CommandLineOptions += "--nogui";
```

Comment:
At the places in Jamulus where I have used the QStringList class, I have used the operator<< instead of the +=. Does the += give a different result? If not, for consistency, we should use << here, too.

Reply:
Oh - I was actually surprised `+=` worked.  My C backgound -- `<<` is very C++ ;).  Yes, `<<` as append seems much better!  I'll fix that now.